blazor/diagram/accessibility.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: Accessibility in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about accessibility in Syncfusion Blazor Diagram component and much more details.
```

Comment:
description: Learn all about accessibility in Syncfusion Blazor Diagram component and many more details.